docs/blogging.rst

```diff
@@ -181,7 +181,7 @@ General writing suggestions
 -  Yes, other people have written about what you have. But no one has *your* voice. So if you feel like writing about it, **write about it**.
 -  Use grammar and punctuation well. 
   
-  - Use `Grammarly <https://www.grammarly.com>`__.
+  - Use `Grammarly <https://www.grammarly.com>`__. [3]_
```

Comment:
Instead, add the note section just after. Every warning level thing should be on the face. 

```
.. note:: Text
```